src/main/java/com/wire/bots/roman/resources/BroadcastResource.java

```diff
@@ -142,6 +106,79 @@ public Response post(@Context ContainerRequestContext context,
         }
     }
 
+    private void broadcastCall(IncomingMessage message, UUID providerId, UUID broadcastId) {
+        for (UUID botId : botsDAO.getBotIds(providerId)) {
+            broadcast.submit(() -> {
+                try {
+                    final UUID messageId = sender.sendCall(message, botId);
+                    if (messageId != null) {
+                        persist(providerId, broadcastId, botId, messageId);
+                    }
+                } catch (Exception e) {
+                    Logger.exception("Broadcast", e);
```

Comment:
I'd probably log the bot ID, the provider id and the broadcast id as well, because they won't be propagated to the logs as this runs inside the thread pool (so no MDC unfortunately)

Reply:
:'(